Changelog for 9.1.4 (#5510)
elastic/fleet-server: docs/release-notes/_snippets/9.1.4/index.md

## 9.1.4 [fleet-server-release-notes-9.1.4]

### Fixes [fleet-server-9.1.4-fixes]

* Reset trace links on bulk items when returning to pool. [#5317](https://github.com/elastic/fleet-server/pull/5317)
* Restore connection limiter. [#5372](https://github.com/elastic/fleet-server/pull/5372)

  Restore connection level limiter to prevent OOM incidents.

>   This limiter is used in addition to the request-level throttle so that once
>   our in-flight requests reaches max_connections a 429 is returned, but if the
>   total connections the server uses is over max_connections*1.1 the server drops
>   the connection before the TLS handshake.

Reviewer: ```suggestion
  This limiter is used in addition to the request-level throttle so that when
  our in-flight requests reaches max_connections, a 429 is returned. If the
  total connections the server uses is over max_connections*1.1 the server drops
  the connection before the TLS handshake.
```

Reviewer: This suggestion makes the content easier to read/parse, but I want to be sure that I did not change the meaning.
